### PR TITLE
[EXT2FS] Ext2Read(): Complete request on VCB_VOLUME_LOCKED case

### PR DESCRIPTION
**Pull request by SergeGautherie**
I checked Read and Write: this is the only case where it is missing.

JIRA issue: [CORE-10645](https://jira.reactos.org/browse/CORE-10645)

Co-authored-by: Victor Martinez Calvo <vicmarcal@...>

### CODE REVIEW

**Comment by JoachimHenze** [comment]
Tried to upstream that yet?
 * PROGRAMMER:       Matt Wu <mattwu@163.com>
 * HOMEPAGE:         http://www.ext2fsd.com

**Comment by SergeGautherie** [comment]
> Tried to upstream that yet?

No, I won't bother.

website: times out.
sourceforge: abandoned years ago.
github: abandoned years ago too.

**Comment by SergeGautherie** [comment]
> PR is stale

Looking for a (active) reviewer...
@JoachimHenze, Could you help, as you asked for this PR?

**Comment by SergeGautherie** [comment]
> PR is stale

Looking for a (active) reviewer...
@JoachimHenze, Could you help, as you asked for this PR?

**Comment by JoachimHenze** [comment]
I just suggested to PR, because Kyle pushed me to it, and I assumed devs with filesystem experience to more likely look at it in a PR than for a JIRA-patch.
I don't want to judge this PR myself. Maybe more experienced devs can judge the correctness better than I can. I do not approve it myself, because I am not aware of how to test it, and I am not aware of any specific bug that it would fix. Most promising would be sending it upstream to allow the maintainers of that module to judge it.
In general it is easier to reach me in chat than with any comment in a PR.

**Review by DarkFire01 — APPROVED** [review]
I use ext2/3 quite a bit with ReactOS.

This project is dead and there's nothing to synch to anymore, it's been replaced and we should make an effort to eventually import the revised version for ext4 perhaps. That all being said I'll merge this now :) as it's simple enough. Ill check with the revived driver and see if it doesn't have this yet so maybe we can reduce the diff.

well first @SergeGautherie  make sure you fix your author email and name sections and then ill merge ;)

**Comment by DarkFire01** [comment]
IMG_4269

The author email is using the auto-generated GitHub one.

**Comment by SergeGautherie** [comment]
> The author email is using the auto-generated GitHub one.

That is a GitHub UI issue, as my commit is correct. (Already discussed in other PR(s).)

**Pull request merged**
